### PR TITLE
Add Bright Mountain Media Bid Adapter

### PR DESCRIPTION
**Pull request by BrightMountainMediaInc**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  "bidder": "brightmountainmedia",
  "params": {
    "placement_id": "5f21784949be81002079d08c",
    "traffic": "banner"
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer [dev@brightmountainmedia.com](mailto:dev@brightmountainmedia.com)
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/brightMountainMediaBidAdapter.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      mediaTypes: {
+          banner: {
+              sizes: [[300, 250]]
+          }
+      },
+      bids: [
+          {
+              bidder: 'brightmountainmedia',
+              params: {
+                  placement_id: 5f21784949be82079d08c,
```
Comment (musikele) [inline modules/brightMountainMediaBidAdapter.md:28]: to be a valid JSON, the placement_id value should be wrapped in quotes: 

```
placement_id: '5f21784949be82079d08c',
```

**Comment by bretg** [comment]
docs PR https://github.com/prebid/prebid.github.io/pull/2220

File: modules/brightMountainMediaBidAdapter.md (hunk 1 above)
Comment (BrightMountainMediaInc) [reply modules/brightMountainMediaBidAdapter.md]: Fixed in 1e9b7dcdd43bf215fada66f1287c4404e03f5032

**Comment by BrightMountainMediaInc** [comment]
> When trying the adapter with hello_world.html test page, I see this in prebid logs:
> 
> ```
> WARNING: Bidder brightmountainmedia made bid for unknown request ID: . Ignoring.
> ```
> 
> If the bidder adds a requestId to the response, even when the bid is empty, the warning would go away.
> 
> Also: did you prepare a PR for the documentation website?

Thanks @bretg  for linking the docs PR.

```
WARNING: Bidder brightmountainmedia made bid for unknown request ID: . Ignoring.
```
`requestId` is now being sent on every response so this warning won't appear anymore.

**Pull request merged**